components/list/list-item-drag-handle.js

```diff
@@ -0,0 +1,196 @@
+
+import '../button/button-icon.js';
+import '../icons/icon.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { buttonStyles } from '../button/button-styles.js';
+import { getFirstFocusableDescendant } from '../../helpers/focus.js';
+
+const keyCodes = Object.freeze({
+	END: 35,
+	HOME: 36,
+	UP: 38,
+	DOWN: 40,
+	SPACE: 32,
+	ENTER: 13,
+	ESC: 27,
+	TAB: 9,
+	LEFT: 37,
+	RIGHT: 39,
+
+});
+
+export const dragActions = Object.freeze({
+	first: 'last',
+	end: 'first',
+	up: 'up',
+	down: 'down',
+	active: 'keyboard-active',
+	save: 'keyboard-deactivate-save',
+	cancel: 'keyboard-deactivate-cancel',
+	nextElement: 'next-element',
+	previousElement: 'previous-element'
+});
+
+class listItemDragHandle extends LitElement {
+
+	static get properties() {
+		return {
+			keyboardActive: { type: Boolean, attribute: 'keyboard-active', reflect: true },
+			text: { type: String }
+		};
+	}
+
+	static get styles() {
+		return [ buttonStyles, css`
+			:host {
+				align-items: center;
+				display: flex;
+				margin: 0.25rem;
+			}
+			:host([hidden]) {
+				display: none;
+			}
+			.d2l-list-item-drag-handle-dragger-button,
+			.d2l-list-item-drag-handle-keyboard-button {
+				display: grid;
+				grid-auto-columns: auto;
```

Comment:
Isn't this the default?

Reply:
I'll double check and if it is i'll remove. It be surprising if it isn't default.